EssentialsChat/src/main/java/com/earth2me/essentials/chat/EssentialsChat.java

```diff
@@ -47,7 +47,6 @@ public void onEnable() {
 
     @Override
     public boolean onCommand(final CommandSender sender, final Command command, final String commandLabel, final String[] args) {
-        metrics.markCommand(command.getName(), true);
         return ess.onCommandEssentials(sender, command, commandLabel, args, EssentialsChat.class.getClassLoader(), "com.earth2me.essentials.chat.Command", "essentials.", null);
```

Comment:
```suggestion
        metrics.markCommand(command.getName(), true);
        return ess.onCommandEssentials(sender, command, commandLabel, args, EssentialsChat.class.getClassLoader(), "com.earth2me.essentials.chat.Command", "essentials.", null);
```

Restore this, it's here for when we introduce a proper metrics system